librz/include/rz_util/ht_inc.h

```diff
@@ -39,14 +39,47 @@
 #define VALUE_TYPE     ut64
 #define KEY_TO_HASH(x) ((ut32)(x))
 #define HT_NULL_VALUE  0
-#else
+#elif HT_TYPE == 4
 #define HtName_(name)  name##PU
 #define Ht_(name)      ht_pu_##name
 #define HT_(name)      HtPU##name
 #define KEY_TYPE       void *
 #define VALUE_TYPE     ut64
 #define KEY_TO_HASH(x) ((ut32)(uintptr_t)(x))
 #define HT_NULL_VALUE  0
+#elif HT_TYPE == 5
```

Comment:
I think it worth probably to add into those a short comment what kind of HT they represent too, similarly to the `ht_XX.h` headers.